boards/arm/frdm_k64f/frdm_k64f_bt.dts

```diff
@@ -0,0 +1,12 @@
+#include <frdm_k64f.dts>
+
+/ {
+	chosen {
+		zephyr,bt-uart = &uart3;
```

Comment:
What is the HW which is being described here?
Is that a kind of shield?

Reply:
in this case its a shield

Reply:
So why not actually do this as a shield in `boards/shields`? There are at least two boards (frdm_k64f and mimxrt1050_evk) that could use it

Reply:
For frdm_k64f & mimxrt1050_evk, what is the "shield" board for bluetooth?  Is this a FRDM-KW41Z or something else?

Reply:
Yes, FRDM-KW41Z can be used either as a base board or as a shield.

Reply:
> Yes, FRDM-KW41Z can be used either as a base board or as a shield.

Right, I guess I was asking, is the assumption that it is specifically a KW41Z as a shield for CONFIG_BT on FRDM-K64F & mimxrt1050_evk.  Or the assumption more generic about some random BT device/shield?